Department-for-Transport-Disruptions/create-disruptions-data
# Fix for the logout issue with incompatible lambda edge runtime for node:crypto module

Testing instructions

- Login as org admin and access the Social media page. Connect to hootsuite should display the details about successful hootsuite connection
